engine/engine.go

```diff
@@ -539,7 +540,7 @@ func containsDuplicateLabelSet(series []labels.Labels) bool {
 	seen := make(map[uint64]struct{}, len(series))
 	for i := range series {
 		buf = buf[:0]
-		h, buf = series[i].HashWithoutLabels(buf)
+		h = xxhash.Sum64(series[i].BytesWithoutLabels(buf))
```

Comment:
Why BytesWithoutLabels instead of Bytes? :thinking:

Reply:
Good point, I completely missed that `Bytes` exists :) Should be updated now.